bundles/persistence/org.openhab.persistence.jpa/pom.xml

```diff
@@ -34,7 +34,7 @@
 			<plugin>
 		        <groupId>org.apache.openjpa</groupId>
 		        <artifactId>openjpa-maven-plugin</artifactId>
-		        <version>2.3.0</version>
+		        <version>2.4.0</version>
```

Comment:
Could you define a maven property for the version number so it will be synchronously updated in all the pom when necessary?